cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -133,6 +135,12 @@ private Instant extractEntryCreationDate() {
             .orElse(null);
     }
 
+    private Instant extractEntryLastModifiedDate() {
+        return Optional.ofNullable(cristinObject.getEntryLastModifiedDate())
+                .map(ld -> ld.atStartOfDay().toInstant(zoneOffset()))
+                .orElse(extractEntryCreationDate());
```

Comment:
use orElseGet(()->) the difference between the two is that orElse(actionHere()) will always execute even the Optional is succesfull. 
This is happenning because when I do foo(boo()) boo is always called before foo and the result of boo's execution is passed to foo. So when you call orElse(boo()), you first calculate the boo() value and then you may or may not use it. 
Instead you can use orElseGet(()->) to pass as a parameter a function that may or  may not be executed